src/appmixer/pipedrive/crm/FindLeads/FindLeads.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+/**
+ * Find Leads action.
+ * @extends {Component}
+ */
+module.exports = {
+    async receive(context) {
+        const generateOutputPortOptions = context.properties.generateOutputPortOptions;
```

Comment:
Add a newline after function declaration.
```suggestion
    async receive(context) {

        const generateOutputPortOptions = context.properties.generateOutputPortOptions;
```